src/azure-cli/azure/cli/command_modules/acr/tests/latest/test_acr_artifact_streaming_commands.py

```diff
@@ -4,16 +4,20 @@
 # --------------------------------------------------------------------------------------------
 
 from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+from azure.cli.testsdk.scenario_tests.decorators import AllowLargeResponse, live_only
 
 
 class AcrArtifactStreamingCommandsTests(ScenarioTest):
 
     @ResourceGroupPreparer()
+    @AllowLargeResponse()
+    @live_only()
```

Comment:
Hi @zhoxing-ms , for this test, we don't know why it would only pass in live mode. After I generate the recording, when I reran the test, it would fail due to this issue, which I mentioned in my previous comments, but received no further explanation. I am not sure if this is related to the test framework or not. We have similar setups for other tests, but we only see this happening for this test. Do you know why or how to fix this? Thanks!

"I am starting to see some error from the test framework for the test test_acr_artifact_streaming .I am getting similar failure as this Open Issue:L https://github.com/Azure/azure-cli/issues/28283

E vcr.errors.CannotOverwriteExistingCassetteException: Can't overwrite existing cassette ('d:\repos\azure-cli\src\azure-cli\azure\cli\command_modules\acr\tests\latest\recordings\test_acr_artifact_streaming.yaml') in your current record mode ('once').
E No match for the request (<Request (POST) [https://cliregoxvpwrdu2hd2qx.azurecr.io/oauth2/exchange>](https://cliregoxvpwrdu2hd2qx.azurecr.io/oauth2/exchange%3E)) was found.
E Found 10 similar requests with 1 different matcher(s) :
E
E 1 - (<Request (POST) [https://clireg000002.azurecr.io/oauth2/exchange>](https://clireg000002.azurecr.io/oauth2/exchange%3E)).
E Matchers succeeded : ['method', 'scheme', 'port', 'path', '_custom_request_query_matcher']
E Matchers failed :
E host - assertion failure :
E cliregoxvpwrdu2hd2qx.azurecr.io != clireg000002.azurecr.io
E
E 2 - (<Request (POST) [https://clireg000002.azurecr.io/oauth2/exchange>](https://clireg000002.azurecr.io/oauth2/exchange%3E)).
E Matchers succeeded : ['method', 'scheme', 'port', 'path', '_custom_request_query_matcher']
E Matchers failed :
E host - assertion failure :
E cliregoxvpwrdu2hd2qx.azurecr.io != clireg000002.azurecr.io
E
E 3 - (<Request (POST) [https://clireg000002.azurecr.io/oauth2/exchange>](https://clireg000002.azurecr.io/oauth2/exchange%3E)).
E Matchers succeeded : ['method', 'scheme', 'port', 'path', '_custom_request_query_matcher']
E Matchers failed :
E host - assertion failure :
E cliregoxvpwrdu2hd2qx.azurecr.io != clireg000002.azurecr.io
E
E 4 - (<Request (POST) [https://clireg000002.azurecr.io/oauth2/exchange>](https://clireg000002.azurecr.io/oauth2/exchange%3E)).
E Matchers succeeded : ['method', 'scheme', 'port', 'path', '_custom_request_query_matcher']
E Matchers failed :
E host - assertion failure :
E cliregoxvpwrdu2hd2qx.azurecr.io != clireg000002.azurecr.io
E
E 5 - (<Request (POST) [https://clireg000002.azurecr.io/oauth2/exchange>](https://clireg000002.azurecr.io/oauth2/exchange%3E)).
E Matchers succeeded : ['method', 'scheme', 'port', 'path', '_custom_request_query_matcher']
E Matchers failed :
E host - assertion failure :
E cliregoxvpwrdu2hd2qx.azurecr.io != clireg000002.azurecr.io
E
E 6 - (<Request (POST) [https://clireg000002.azurecr.io/oauth2/exchange>](https://clireg000002.azurecr.io/oauth2/exchange%3E)).
E Matchers succeeded : ['method', 'scheme', 'port', 'path', '_custom_request_query_matcher']
E Matchers failed :
E host - assertion failure :
E cliregoxvpwrdu2hd2qx.azurecr.io != clireg000002.azurecr.io
E
E 7 - (<Request (POST) [https://clireg000002.azurecr.io/oauth2/exchange>](https://clireg000002.azurecr.io/oauth2/exchange%3E)).
E Matchers succeeded : ['method', 'scheme', 'port', 'path', '_custom_request_query_matcher']
E Matchers failed :
E host - assertion failure :
E cliregoxvpwrdu2hd2qx.azurecr.io != clireg000002.azurecr.io
E
E 8 - (<Request (POST) [https://clireg000002.azurecr.io/oauth2/exchange>](https://clireg000002.azurecr.io/oauth2/exchange%3E)).
E Matchers succeeded : ['method', 'scheme', 'port', 'path', '_custom_request_query_matcher']
E Matchers failed :
E host - assertion failure :
E cliregoxvpwrdu2hd2qx.azurecr.io != clireg000002.azurecr.io
E
E 9 - (<Request (POST) [https://clireg000002.azurecr.io/oauth2/exchange>](https://clireg000002.azurecr.io/oauth2/exchange%3E)).
E Matchers succeeded : ['method', 'scheme', 'port', 'path', '_custom_request_query_matcher']
E Matchers failed :
E host - assertion failure :
E cliregoxvpwrdu2hd2qx.azurecr.io != clireg000002.azurecr.io
E
E 10 - (<Request (POST) [https://clireg000002.azurecr.io/oauth2/exchange>](https://clireg000002.azurecr.io/oauth2/exchange%3E)).
E Matchers succeeded : ['method', 'scheme', 'port', 'path', '_custom_request_query_matcher']
E Matchers failed :
E host - assertion failure :
E cliregoxvpwrdu2hd2qx.azurecr.io != clireg000002.azurecr.io

env\lib\site-packages\vcr\stubs_init_.py:263: CannotOverwriteExistingCassetteException"

Reply:
Yes, this issue seems unrelated to your PR~ But I'm sorry, as I've been busy recently, I don't have enough time and energy to delve deeper into it at the moment. 
This test issue do not block the merge of this PR. We can merge this PR first and then optimize this test issue in subsequent PRs in the feature.

Reply:
Because you've recorded the random name `cliregoxvpwrdu2hd2qx` in your test recordings. It somehow wasn't replaced by random name replacer in test framework.

For short term solution, you can manually replace all `cliregoxvpwrdu2hd2qx` with `clireg000002` in your test recordings.

For long term solution, you need to figure out why the random name replacer doesn't work for your test and fix it.